concepticondata/conceptlists/Healey-1970-364.tsv

```diff
@@ -0,0 +1,365 @@
+Healey-1970-364-3	3	anus	1935	ANUS
+Healey-1970-364-4	4	arm	2121	ARM OR HAND
+Healey-1970-364-5	5	armpit	1886	ARMPIT
+Healey-1970-364-6	6	arrow	977	ARROW
+Healey-1970-364-7	7	ash	646	ASH
+Healey-1970-364-8	8	ask for/about	1309	ASK (INQUIRE)
+Healey-1970-364-9	9	at	1461	AT
+Healey-1970-364-10	10	avenge		
+Healey-1970-364-11	11	axe, stone	677	AXE
+Healey-1970-364-12	12	back	1291	BACK
+Healey-1970-364-13	13	backbone	805	SPINE
+Healey-1970-364-14	14	bad	1292	BAD
+Healey-1970-364-15	15	bake	183	BAKE
+Healey-1970-364-16	16	banana	868	BANANA
+Healey-1970-364-17	17	bat, fruit	235	FLYING FOX
+Healey-1970-364-18	18	bathe	138	BATHE
+Healey-1970-364-19	19	beard	678	BEARD
+Healey-1970-364-20	20	belly	1251	BELLY
+Healey-1970-364-21	21	beneath	1485	BELOW OR UNDER
+Healey-1970-364-22	22	bile	1780	BILE
+Healey-1970-364-23	23	bird	937	BIRD
+Healey-1970-364-24	24	bird of paradise	2673	BIRD OF PARADISE
+Healey-1970-364-25	25	birth, give	1195	GIVE BIRTH
+Healey-1970-364-26	26	bite	1403	BITE
+Healey-1970-364-27	27	black	163	BLACK
+Healey-1970-364-28	28	blood	946	BLOOD
+Healey-1970-364-29	29	body	1480	BODY
+Healey-1970-364-30	30	boil, a	793	BOIL (OF SKIN)
+Healey-1970-364-31	31	bone	1394	BONE
+Healey-1970-364-32	32	bow	994	BOW
+Healey-1970-364-33	33	brain	1583	BRAIN
+Healey-1970-364-34	34	branch	1531	BRANCH
+Healey-1970-364-35	35	breadfruit	2371	BREADFRUIT
+Healey-1970-364-36	36	break (rope)	460	BREAK (OF ROPE)
+Healey-1970-364-37	37	break (stone)	2558	BREAK (CLEAVE)
+Healey-1970-364-38	38	break (wood)		
+Healey-1970-364-39	39	breaks, (rope)	460	BREAK (OF ROPE)
+Healey-1970-364-40	40	breaks, (stone)		
+Healey-1970-364-41	41	breast	1402	BREAST
+Healey-1970-364-42	42	bring	692	BRING
+Healey-1970-364-43	43	brother, elder -	1759	OLDER BROTHER
+Healey-1970-364-44	44	brother, younger	1760	YOUNGER BROTHER
+Healey-1970-364-45	45	burn	2102	BURN
+Healey-1970-364-46	46	butterfly	1791	BUTTERFLY
+Healey-1970-364-47	47	buttocks	834	BUTTOCKS
+Healey-1970-364-48	48	call	1084	CALL
+Healey-1970-364-49	49	canoe	1970	CANOE
+Healey-1970-364-50	50	carry	700	CARRY
+Healey-1970-364-51	51	centipede	1974	CENTIPEDE
+Healey-1970-364-52	52	cheek	1730	CHEEK
+Healey-1970-364-53	53	child	2099	CHILD
+Healey-1970-364-54	54	chin	1510	CHIN
+Healey-1970-364-55	55	chop	155	CHOP
+Healey-1970-364-56	56	chop down	463	FELL (A TREE)
+Healey-1970-364-57	57	climb	1102	CLIMB
+Healey-1970-364-58	58	coconut	970	COCONUT
+Healey-1970-364-59	59	cold	1287	COLD
+Healey-1970-364-60	60	come	1446	COME
+Healey-1970-364-61	61	command	1998	COMMAND
+Healey-1970-364-62	62	cooked, be	269	COOKED
+Healey-1970-364-63	63	copulate	549	COPULATE
+Healey-1970-364-64	64	cough	879	COUGH
+Healey-1970-364-65	65	crocodile	1857	CROCODILE
+Healey-1970-364-66	66	crown-pigeon		
+Healey-1970-364-67	67	cry	1839	CRY
+Healey-1970-364-68	68	cut	1432	CUT
+Healey-1970-364-69	69	dark	706	DARK
+Healey-1970-364-70	70	daughter	1357	DAUGHTER
+Healey-1970-364-71	71	daylight	1225	DAY (NOT NIGHT)
+Healey-1970-364-72	72	dead person	1625	DEAD
```

Comment:
I'd also unmapped because the concept on the list is more specific and because of previous mappings.